README.md

```diff
@@ -106,9 +106,9 @@ Don't forget the `.` at the end!
 
 ## Documentation
 
-Documentation along with a quick start guide can be found in the [docs/](/docs) directory.
+Documentation, along with the Quick Start Guide, can be found in the [docs/](/docs) directory.
```

Comment:
what's the reason for this change/capitalization?

Reply:
From what I gathered, that is the current title of the specified document (albeit without the stylized "tinygrad").